assets/css/structures/_table-of-content.scss

```diff
@@ -1,3 +1,9 @@
 .toc__list-item {
   break-inside: avoid;
+
+    a {
+    display: inline-block;
```

Comment:
The `inline-block` here causes an issue with the underline for words that become multiple lines, like OCD:

<img width="310" alt="Obsessive Compulsive Disorder has underline that extends beyond the word" src="https://user-images.githubusercontent.com/7484549/74808755-b0a6fc00-533f-11ea-95f9-1616638e5bf9.png">